apiserver/clusterservice/clusterimpl.go

```diff
@@ -739,13 +739,14 @@ func CreateCluster(request *msgs.CreateClusterRequest, ns, pgouser string) msgs.
 			}
 		}
 
-		//verify that for autofail clusters we have a replica requested
+		// include a warning in the response indicating that autofail capabilities will not be able to be utilized
+		// until at least one replica is added to the cluster 
 		if newInstance.Spec.Replicas == "0" {
-			if request.AutofailFlag || apiserver.Pgo.Cluster.Autofail {
-				resp.Status.Code = msgs.Error
-				resp.Status.Msg = "replica-count of 1 or more is required when autofail is specified"
-				return resp
+			noReplicasResult := "Warning: Please add at least one replica to utilize autofail functionality"
```

Comment:
I will go ahead and remove (it was more that a user couldn't create a single node cluster with autofail enabled in the past that I was concerned could cause confusion since they are now able to).

Reply:
:+1: